Explorer/Assets/DCL/AvatarRendering/Emotes/Systems/Play/CharacterEmoteSystem.cs

```diff
@@ -243,6 +244,13 @@ private void CleanUp(Profile profile, in DeleteEntityIntention deleteEntityInten
                 messageBus.OnPlayerRemoved(profile.UserId);
         }
 
+        [Query]
+        private void DisableCharacterController(ref CharacterController characterController, in CharacterEmoteComponent emoteComponent)
+        {
+            bool isPlayingEmote = emoteComponent.CurrentAnimationTag == AnimationHashes.EMOTE || emoteComponent.CurrentAnimationTag == AnimationHashes.EMOTE_LOOP;
```

Comment:
nice check, I was using `EmoteReference != null` for that, but your approach looks better

Reply:
maybe you can move this check inside `CharacerEmoteComponent`, because I have asmdef issues when referencing `AnimationHashes` in one of my scripts

Reply:
you can move `AnimationHashes.cs` into `Assets/DCL/Character/CharacterMotion/Animation`

and then replace all `.CurrentEmoteReference != null` checks that are used as a check of currently played emote (maybe leaving in comment that this was used before)